<!DOCTYPE html>
<html>

  

    <head>
 <meta charset="utf-8">
<title>Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context</title>
<meta name="viewport" content="width=device-width">
<meta name="description" content="JKU Visual Data Science Lab.">
<link rel="canonical" href="http://localhost:4000/publications/2010_pacificvis_caleydo/">

<!-- Custom Fonts -->
<link href='//fonts.googleapis.com/css?family=Lato:300,400,700,900,300italic,400italic,700italic,900italic|Patua+One|Vollkorn:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
<link href='//fonts.googleapis.com/css?family=PT+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>

<!-- Custom CSS -->
<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
<link rel="stylesheet" href="/css/site.css">

<!-- Fav and touch icons -->
<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
<link rel="apple-touch-icon" href="/favicon.png" />

<link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/jquery.slick/1.3.6/slick.css" />


 <!-- metadata for link preview  -->
  <!-- Schema.org markup for Google+ -->
  <meta itemprop="name" content="Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context">
  <meta itemprop="description" content=" The goal of our work is to support experts in the process of hypotheses generation concerning the roles of genes in diseases. For a deeper understanding of the complex interdependencies between genes, it is important to bring gene expressions (measurements) into context with pathways. Pathways, which are models of biological processes, are available in online databases. In these databases, large networks are decomposed into small sub-graphs for better manageability. This simplification results in a loss of context, as pathways are interconnected and genes can occur in multiple instances scattered over the network. Our main goal is therefore to present all relevant information, i.e., gene expressions, the relations between expression and pathways and between multiple pathways in a simple, yet effective way. To achieve this we employ two different multiple-view approaches. Traditional multiple views are used for large datasets or highly interactive visualizations, while a 2.5D technique is employed to support a seamless navigation of multiple pathways which simultaneously links to the expression of the  contained genes. This approach facilitates the understanding of the interconnection of pathways, and enables a non-distracting relation to gene expression data. We evaluated Caleydo with a group of users from the life science community. Users were asked to perform three tasks: pathway exploration, gene expression analysis and information comparison with and without visual links, which had to be conducted in four different conditions. Evaluation results show that the system can improve the process of understanding the complex network of pathways and the individual effects of gene expression regulation considerably. Especially the quality of the available contextual information and the spatial organization was rated good for the presented 2.5D setup.">
  

  <!-- Twitter Card data -->
  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@marc_streit">
  <meta name="twitter:title" content="Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context">
  <meta name="twitter:description" content=" The goal of our work is to support experts in the process of hypotheses generation concerning the roles of genes in diseases. For a deeper understanding of the complex interdependencies between genes, it is important to bring gene expressions (measurements) into context with pathways. Pathways, which are models of biological processes, are available in online databases. In these databases, large networks are decomposed into small sub-graphs for better manageability. This simplification results in a loss of context, as pathways are interconnected and genes can occur in multiple instances scattered over the network. Our main goal is therefore to present all relevant information, i.e., gene expressions, the relations between expression and pathways and between multiple pathways in a simple, yet effective way. To achieve this we employ two different multiple-view approaches. Traditional multiple views are used for large datasets or highly interactive visualizations, while a 2.5D technique is employed to support a seamless navigation of multiple pathways which simultaneously links to the expression of the  contained genes. This approach facilitates the understanding of the interconnection of pathways, and enables a non-distracting relation to gene expression data. We evaluated Caleydo with a group of users from the life science community. Users were asked to perform three tasks: pathway exploration, gene expression analysis and information comparison with and without visual links, which had to be conducted in four different conditions. Evaluation results show that the system can improve the process of understanding the complex network of pathways and the individual effects of gene expression regulation considerably. Especially the quality of the available contextual information and the spatial organization was rated good for the presented 2.5D setup.">
  <meta name="twitter:creator" content="@marc_streit">
  

  <!-- Open Graph data -->
  <meta property="og:title" content="Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://localhost:4000/publications/2010_pacificvis_caleydo/" />
  <meta property="og:description" content=" The goal of our work is to support experts in the process of hypotheses generation concerning the roles of genes in diseases. For a deeper understanding of the complex interdependencies between genes, it is important to bring gene expressions (measurements) into context with pathways. Pathways, which are models of biological processes, are available in online databases. In these databases, large networks are decomposed into small sub-graphs for better manageability. This simplification results in a loss of context, as pathways are interconnected and genes can occur in multiple instances scattered over the network. Our main goal is therefore to present all relevant information, i.e., gene expressions, the relations between expression and pathways and between multiple pathways in a simple, yet effective way. To achieve this we employ two different multiple-view approaches. Traditional multiple views are used for large datasets or highly interactive visualizations, while a 2.5D technique is employed to support a seamless navigation of multiple pathways which simultaneously links to the expression of the  contained genes. This approach facilitates the understanding of the interconnection of pathways, and enables a non-distracting relation to gene expression data. We evaluated Caleydo with a group of users from the life science community. Users were asked to perform three tasks: pathway exploration, gene expression analysis and information comparison with and without visual links, which had to be conducted in four different conditions. Evaluation results show that the system can improve the process of understanding the complex network of pathways and the individual effects of gene expression regulation considerably. Especially the quality of the available contextual information and the spatial organization was rated good for the presented 2.5D setup." />
  



</head>


      <body>
      <div class="wrap">
        <div class="navigation">
    <div class="logo">
        <a href="/">
            <!-- JKU Visual Data Science Lab -->
            <img class="logo-header-svg" src="/assets/images/logos/jku-vds-lab-logo.svg" alt="JKU Visual Data Science Lab - Logo">
        </a>
    </div>

    <div class="logo-subtitle">Part of the <nobr><a href="https://www.jku.at/en/institute-of-computer-graphics/" target="_blank">Institute of Computer Graphics</a></nobr> @ <nobr><a href="https://www.jku.at/" target="_blank">Johannes Kepler University Linz</a></nobr>, Austria</div>


    <div class="navigation-buttons"> 
        <ul>
            <li class="home-button">
                <a href="/"><span class="nav-icon"><i class="fa fa-home"></i></span><span class="nav-text">Home</span></a>
            </li>
            <li class="team-button">                
                <a href="/team/"><span class="nav-icon"><i class="fa fa-group"></i></span><span class="nav-text">Team</span></a>
            </li>
            <li class="publication-button">
                <a href="/publications/"><span class="nav-icon"><i class="fa fa-book"></i></span><span class="nav-text">Publications</span></a>
            </li>
            <li class="teaching-button">
                <a href="https://www.jku.at/en/institute-of-computer-graphics/teaching/" target="_blank"><span class="nav-icon" style="vertical-align: sub;"><img src="/assets/images/icons/icon-teaching.svg" height="15"></span><span class="nav-text">Teaching</span></a>
            </li>
            <li class="location-button">
                <a href="https://www.google.com/maps/place/Science+Park+3/@48.3348119,14.322509,17.52z/data=!4m12!1m6!3m5!1s0x477398434ec8e84d:0xe0d1670c08989881!2sJohannes+Kepler+Universit%C3%A4t+Linz+-+JKU!8m2!3d48.336614!4d14.319305!3m4!1s0x4773991d5880f437:0xc58d7ca1cdc2b04!8m2!3d48.3351119!4d14.3242242" target="_blank"><span class="nav-icon"><i class="fa fa-map-marker"></i></span><span class="nav-text">Location</span></a>
            </li>

            <!-- <li class="home-button">
                <a href="/"><span class="nav-icon"><i class="fa fa-home"></i></span><span class="nav-text">Home</span></a>
            </li>
            <li class="team-button">
                <a href="/team/"><span class="nav-icon"><i class="fa fa-user"></i></span><span class="nav-text">Team</span></a>
            </li>
            <li class="publication-button">
                <a href="/publications/"><span class="nav-icon"><i class="fa fa-book"></i></span><span class="nav-text">Publications</span></a>
            </li>
            <li class="teaching-button">
                <a href="https://www.jku.at/en/institute-of-computer-graphics/teaching/" target="_blank"><span class="nav-icon"><i class="fa fa-group"></i></span><span class="nav-text">Teaching</span></a>
            </li> -->
            

        </ul>
    </div>

</div>


        <div class="publication">

  <header class="publication-title">
    <h2>Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context</h2>
  </header>

  

  


  <article class="publication-content-main">

    


    
    <h2>Abstract</h2>
    <p> <p>The goal of our work is to support experts in the process of hypotheses generation concerning the roles of genes in diseases. For a deeper understanding of the complex interdependencies between genes, it is important to bring gene expressions (measurements) into context with pathways. Pathways, which are models of biological processes, are available in online databases. In these databases, large networks are decomposed into small sub-graphs for better manageability. This simplification results in a loss of context, as pathways are interconnected and genes can occur in multiple instances scattered over the network. Our main goal is therefore to present all relevant information, i.e., gene expressions, the relations between expression and pathways and between multiple pathways in a simple, yet effective way. To achieve this we employ two different multiple-view approaches. Traditional multiple views are used for large datasets or highly interactive visualizations, while a 2.5D technique is employed to support a seamless navigation of multiple pathways which simultaneously links to the expression of the  contained genes. This approach facilitates the understanding of the interconnection of pathways, and enables a non-distracting relation to gene expression data. We evaluated Caleydo with a group of users from the life science community. Users were asked to perform three tasks: pathway exploration, gene expression analysis and information comparison with and without visual links, which had to be conducted in four different conditions. Evaluation results show that the system can improve the process of understanding the complex network of pathways and the individual effects of gene expression regulation considerably. Especially the quality of the available contextual information and the spatial organization was rated good for the presented 2.5D setup.</p></p>
    <br>
    

    <h2>Citation</h2>
    <div class="citation">
      <p>
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        Alexander Lex, 
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <a href="/persons/streit/">Marc
          Streit</a>, 
        
        <!-- <a href="https://marc-streit.com">Marc&nbsp;Streit</a>,  -->
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        Ernst Kruijff, 
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        Dieter Schmalstieg
        
        
        


        <br />
        <span class="paper-title">
          
          <b><a href="http://data.jku-vds-lab.at/papers/2010_pacificvis_caleydo.pdf">Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context</a></b>
          
        </span><br />
        

        <span class="paper-citation">
          
          
          



          Proceedings of IEEE Pacific Visualization Symposium (PacificVis '10), 
          57--64,   <a
            href="https://dx.doi.org/10.1109/PACIFICVIS.2010.5429609">doi:10.1109/PACIFICVIS.2010.5429609</a>, 2010.
          

          

          
          <p class="note" style="font-style:italic;">
            <b>Note:</b> Extended version of 2009 Oxford Bioinformatics Applications Note
          </p>
          
        </span>
    </div>

    
    <h2>BibTeX</h2>
    
    <!-- here replacing the curly brackets with html special characters. { is &#123; ; } is &#125; -->
    <!-- formatting will mess up the way it is displayed on the page -->
<pre class="bibtex">
@inproceedings&#123;2010_pacificvis_caleydo,
    title = &#123;Caleydo: Design and Evaluation of a Visual Analysis Framework for Gene Expression Data in its Biological Context&#125;,
    author = &#123;Alexander Lex and Marc Streit and Ernst Kruijff and Dieter Schmalstieg&#125;,
    booktitle = &#123;Proceedings of IEEE Pacific Visualization Symposium (PacificVis '10)&#125;,
    doi = &#123;10.1109/PACIFICVIS.2010.5429609&#125;,
    pages = &#123;57--64&#125;,
    year = &#123;2010&#125;
}
</pre>
    
    <br>
    <!-- main markdown content -->
    

  </article>

  <article class="publication-content-aside">
    

    


    <h2>Resources</h2>
    <ul>
      
      <li>
        <a href="https://dx.doi.org/10.1109/PACIFICVIS.2010.5429609" target="_blank">
          <p>
            <i class="fa fa-globe fa-lg resources-icon" aria-hidden="true"></i>
            Publisher
          </p>
        </a>
      </li>
      

      

      
      <li>
        <a href="http://data.jku-vds-lab.at/papers/2010_pacificvis_caleydo.pdf" target="_blank">
          <p>
            <i class="fa fa-download fa-lg resources-icon" aria-hidden="true"></i>
            Publication (PDF)
          </p>
        </a>
      </li>
      
      
      

      
    </ul>
    <!-- <ul>
      
      <li>
        <i class="fa fa-globe"></i> &nbsp;<a href="https://dx.doi.org/10.1109/PACIFICVIS.2010.5429609">Publisher</a>
      </li>
      

      

      
      <li>
        <i class="fa fa-download"></i> &nbsp;<a href="http://data.jku-vds-lab.at/papers/2010_pacificvis_caleydo.pdf">Publication (PDF)</a>
      </li>
      
      
      

      
    </ul>
     -->

    

    

    

    <!-- (deprecated) for old versions, where you had to create a video file -->
    <!-- 
      <h2>Video(s)</h2>
     -->
    
    
    
    
    
    
    <div class="video">
      <!-- <p><b>Caleydo Bucket Pacificvis Demonstration Video</b></p> -->
      
      <div class="video-wrapper">
        <iframe src="//www.youtube.com/embed/8SV3Id_lvNY?controls=2&showinfo=0&iv_load_policy=3"
          frameborder="0" allowfullscreen></iframe>
      </div>
      
      <!-- 
      <p><i class="fa fa-download"></i> &nbsp;<a href="http://data.jku-vds-lab.at/papers/2010_pacificvis_caleydo.avi">Download Video</a>
        <p />
         -->
    </div>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    

  </article>


  <article class="publication-meta">

  </article>

  <article class="publication-resources">
  </article>
</div>


        <footer>
  <div class="footer-content">
    <div class="footer-list-social">
      <ul>
          
          <li>
            <a href="mailto:contact@jku-vds-lab.at">
              <!-- <i class="fa fa-lg fa-envelope-o"></i> -->
              <img class="footer-icon" src="/assets/images/icons/icon-mail.svg" alt="contact@jku-vds-lab.at" title="Email">
            </a>
          </li>
          

          
            <li>
              <a href="https://www.jku.at/en/institute-of-computer-graphics/">
                <img class="footer-icon" class="institution-logo" src="/assets/images/icons/icon-icg.svg" alt="https://www.jku.at/en/institute-of-computer-graphics/" title="Institute of Computer Graphics">
              </a>
            </li>
          
            <li>
              <a href="https://www.jku.at">
                <img class="footer-icon" class="institution-logo" src="/assets/images/icons/icon-jku.svg" alt="https://www.jku.at" title="Johannes Kepler University Linz">
              </a>
            </li>
          

          
          
          <li>
            <a href="https://github.com/jku-vds-lab">
              <!-- <i class="fa fa-lg fa-github"></i>
              <span class="footer-text">GitHub</span> -->
              <img class="footer-icon" src="/assets/images/icons/icon-github.svg" alt="jku-vds-lab" title="GitHub">
            </a>
          </li>
          

          
          <li>
<!--             <a href="https://www.youtube.com/user/UCH0ztMDNtQuIx0-ZWIavG0Q"> TODO: change to this ASAP -->
            <a href="https://www.youtube.com/channel/UCH0ztMDNtQuIx0-ZWIavG0Q">
              <!-- <i class="fa fa-lg fa-youtube"></i>
              <span class="footer-text">YouTube</span> -->
              <img class="footer-icon" src="/assets/images/icons/icon-youtube.svg" alt="UCH0ztMDNtQuIx0-ZWIavG0Q" title="YouTube">
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/marc_streit">
              <!-- <i class="fa fa-lg fa-twitter"></i>
              <span class="footer-text">Twitter</span> -->
              <img class="footer-icon" src="/assets/images/icons/icon-twitter.svg" alt="marc_streit" title="Twitter">
            </a>
          </li>
          

      </ul>
    </div>
  </div>

  <!-- <div class="footer-content">
    <div class="footer-list-institutions">
      <ul>
        
          <li>
            <a href="https://www.jku.at/en/institute-of-computer-graphics/"><img class="institution-logo" src="/assets/images/icons/icon-icg.svg" alt="https://www.jku.at/en/institute-of-computer-graphics/"></a>
          </li>
        
          <li>
            <a href="https://www.jku.at"><img class="institution-logo" src="/assets/images/icons/icon-jku.svg" alt="https://www.jku.at"></a>
          </li>
        
      </ul>
    </div>
 </div>
 -->

  <div class="footer-content">
    <div class="footer-disclaimer">
      <p>
        Copyright &copy; 2021 JKU Visual Data Science Lab. All content on this website is licensed under the <br />
        <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons Attribution license (CC BY)</a>. Copyright of the linked papers is with the publishers.
      </p>
    </div>

  </div>

</footer>

      </div>

      
      <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-XB7C1C7YVE"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', "G-XB7C1C7YVE");
</script>
       
    </body>
    
  
  
</html>